ShowPot/ShowPot/Application/SceneDelegate.swift

```diff
@@ -25,35 +26,45 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         let coordinator = AppCoordinator(navigationController: navigationController)
         coordinator.start()
     }
-
+    
     func sceneDidDisconnect(_ scene: UIScene) {
         // Called as the scene is being released by the system.
         // This occurs shortly after the scene enters the background, or when its session is discarded.
         // Release any resources associated with this scene that can be re-created the next time the scene connects.
         // The scene may re-connect later, as its session was not necessarily discarded (see `application:didDiscardSceneSessions` instead).
     }
-
+    
     func sceneDidBecomeActive(_ scene: UIScene) {
         // Called when the scene has moved from an inactive state to an active state.
         // Use this method to restart any tasks that were paused (or not yet started) when the scene was inactive.
     }
-
+    
     func sceneWillResignActive(_ scene: UIScene) {
         // Called when the scene will move from an active state to an inactive state.
         // This may occur due to temporary interruptions (ex. an incoming phone call).
     }
-
+    
     func sceneWillEnterForeground(_ scene: UIScene) {
         // Called as the scene transitions from the background to the foreground.
         // Use this method to undo the changes made on entering the background.
     }
-
+    
     func sceneDidEnterBackground(_ scene: UIScene) {
         // Called as the scene transitions from the foreground to the background.
         // Use this method to save data, release shared resources, and store enough scene-specific state information
         // to restore the scene back to its current state.
     }
-
-
+    
+    func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
+        
+        // MARK: 카카오톡 소셜로그인 시 서비스 앱으로 돌아왔을 때 카카오 로그인 처리를 정상적으로 완료하기위한 로직
+        // 참고하면 좋은 사이트 : https://developers.kakao.com/docs/latest/ko/kakaologin/ios
+        if let url = URLContexts.first?.url {
+            if (AuthApi.isKakaoTalkLoginUrl(url)) {
+                _ = AuthController.handleOpenUrl(url: url)
+            }
+        }
+    }
+    
 }
```

Comment:
해당 주석은 `if` 구문이 아니라 메서드 정의부 (`func scene( ... `) 부분 위쪽에 올라가는게 맞을 것 같습니다.

그리고 다음 작업부터는 코드 작동 방식이나 로직 관련 설명에는 `///` 또는 `/** */` 주석을 이용해주세요.
```suggestion
    /**
     카카오톡 소셜로그인 시 서비스 앱으로 돌아왔을 때 카카오 로그인 처리를 정상적으로 완료하기위한 로직
     
     - Note: 카카오 로그인 iOS 공식문서 - [카카오톡으로 로그인을 위한 설정](https://developers.kakao.com/docs/latest/ko/kakaologin/ios#before-you-begin-setting-for-kakaotalk)
     */
    func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
        if let url = URLContexts.first?.url {
            if (AuthApi.isKakaoTalkLoginUrl(url)) {
                _ = AuthController.handleOpenUrl(url: url)
            }
        }
    }
    
}
```

Reply:
![image](https://github.com/YAPP-Github/24th-App-Team-3-iOS/assets/43776784/0722d3d1-4311-406a-b21a-3f1387574581)
참고로 저렇게 주석달면 이런식으로 열람이 가능합니다. 추후에 문서 추출이 가능하기도 하구요